website/docs/docs/cloud/secure/ip-restrictions.md

```diff
@@ -39,7 +39,9 @@ To add an IP to the allowlist, from the **IP Restrictions** page:
 	- You can add multiple ranges in the same rule.
 6. Click **Save**
 
-Note that simply adding the IP Ranges will not enforce IP restrictions. For more information, see the section “Enabling Restrictions.”
+Add multiple IP ranges by clicking the **Add IP range** button to create a new text field.
+
+Note that simply adding the IP Ranges will not enforce IP restrictions. For more information, see the [Enabling restrictions](#enabling-restrictions) section.
```

Comment:
Hiya @mirnawong1 

This is a non-blocking suggestion, but do you think it'd be worth adding a sentence advising why adding an IP range won't enforce IP restrictions?

Kind Regards